gnovm/pkg/gnolang/store.go

```diff
@@ -32,8 +30,12 @@ type PackageInjector func(store Store, pn *PackageNode)
 // NativeStore is a function which can retrieve native bodies of native functions.
 type NativeStore func(pkgName string, name Name) func(m *Machine)
 
+// Store is the central interface that specifies the communications between the
+// GnoVM and the underlying data store; currently, generally the Gno.land
+// blockchain, or the file system.
 type Store interface {
 	// STABLE
+	BeginTransaction(baseStore, iavlStore store.Store) TransactionStore
```

Comment:
It is my genuine belief that transaction-like interfaces that implement begin, should also implement commit / end

Reply:
Which doesn't seem to be the case here, anymore

Reply:
So, you're suggesting renaming "Write" to "Commit"?

It's fine to me, I guess I picked Write mostly because (1) it's what we are using for the "cache wrappers" in tm2 and (2) I wanted it to not be related to a notion of "commit" in tm2 (with a corresponding change in AppHash). But if you believe Commit is a better choice here, I'm happy to go with it